oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java

```diff
@@ -55,7 +60,12 @@ public String getConnectionString() {
 
   @Name(OracleConstants.CONNECTION_TYPE)
   @Description("Whether to use an SID or Service Name when connecting to the database.")
-  public String connectionType;
+  private String connectionType;
+
+  @Name(OracleConstants.ROLE)
+  @Description("The login role of the user when connecting to the database.")
+  @Nullable
+  private String role;
```

Comment:
there is a oracle source md file, can you also add this doc to that file?

Reply:
done :)